hosting/src/core/models/event.ts

```diff
@@ -0,0 +1,20 @@
+import * as firebase from 'firebase';
+
+import { GooglePlace } from './google-place-find';
+
+export interface Event {
+    uid: string;
+    type: 'stream' | 'meetup' | 'conference' | 'hackathon' | 'workshop';
```

Comment:
Should probably move this to a type to be used in multiple places

Reply:
ill let you clean up the model sometime ;)

Reply:
Sure 👍